accounts/management/commands/crawler.py

```diff
@@ -101,6 +101,7 @@ async def main(
                                 statuses_count=account["statuses_count"],
                                 note=account["note"],
                                 url=account["url"],
+                                activitypub_id=account["uri"],
```

Comment:
Looks like some older instances don't provide the uri field so we'll have to make it optional here to not crash the whole script:


```suggestion
                                activitypub_id=account.get("uri"),
```

Some examples both on 4.1.25:
 * https://mstdn.jp/
 * https://mastodon.sdf.org/